docker-compose.yml

```diff
@@ -41,7 +41,7 @@ queuerunner:
 mailcatcher:
   image: schickling/mailcatcher
 
-php:  
+php:
   image: spira/docker-hhvm
 #  image: spira/docker-phpfpm
   volumes_from:
```

Comment:
The PHP container also needs to be linked to `.secret.env` as devtools isn't used when the api is actually executed (outside of phpunit).